decryptor/mysql/response_proxy.go

```diff
@@ -295,8 +295,8 @@ func (handler *MysqlHandler) isFieldToDecrypt(field *ColumnDescription) bool {
 func (handler *MysqlHandler) processTextDataRow(rowData []byte, fields []*ColumnDescription) ([]byte, error) {
 	var err error
 	var value []byte
-	var pos int = 0
-	var n int = 0
```

Comment:
hate golang for preference of the implicit explicit ))

Reply:
agree 100%